libs/feature/search/src/lib/utils/service/fields.ts

```diff
@@ -69,15 +75,22 @@ export class SimpleSearchField implements AbstractSearchField {
       })
     )
   }
-  getFiltersForValues(values: FieldValue[]): Observable<FieldFilters> {
+  getFiltersForValues(
+    values: FieldValue[],
+    esFieldName: string = this.esFieldName
```

Comment:
If you change the signature, then you don't correctly implements the abstract class.
BTW, `implements` an abstract is javascript abusive, one should `extend` a class, and `implement` an interface. It would have rise a warning there.